src/app/content/highlights/hooks/receiveDeleteHighlight.spec.ts

```diff
@@ -63,17 +61,41 @@ describe('receiveDeleteHighlight', () => {
     const deleteHighlightClient = jest.spyOn(helpers.highlightClient, 'deleteHighlight')
       .mockRejectedValue(error);
 
-    hook(receiveDeleteHighlight(highlight as unknown as Highlight, meta));
-    await Promise.resolve();
+    try {
+      await hook(receiveDeleteHighlight(highlight as unknown as Highlight, meta));
+    } catch (error) {
+      expect(deleteHighlightClient).toHaveBeenCalled();
```

Comment:
there is no assumption on error type

Reply:
I wanted to test `dispatch(createHighlight(...)` first and then error throwing.
```
} catch (error) {
   dispatch(createHighlight(
      {
          ...payload as unknown as NewHighlight,
          id: payload.id,
       },
       {...meta, revertingAfterFailure: true}
   ));
   
   if (error instanceof CustomApplicationError) {
      throw error;
   }
   
   throw new HighlightDeleteError({ destination });
}
```